### PR TITLE
andniven_sdo.py - added informal constants to clarify parse function'…

### PR DESCRIPTION
**Pull request by TheDragonMask**
…s conditional message processing

### CODE REVIEW

**Review by Boneill3 — COMMENTED** [review]
I was a little confused by max_plus_1 when I first saw it, though it is actually a perfect description. I think this could be simplified using a range like SDO_TX_RANGE = range(0x580, 0x600)

Also, we I think these constants could be moved to the utilities.py

**Comment by Boneill3** [comment]
Correction, check out the message type in can/message.py it actually already contains all of the ranges already so we should be able to just leverage that here!

**Review by Boneill3 — CHANGES_REQUESTED** [review]
Just testing using the request changes option, my comments are already above.

File: canopen_monitor/parse/sdo.py [hunk 1]
```diff
@@ -880,19 +880,24 @@ def __init__(self):
         self.__last_sequence = 0
         self.__awaiting_conf = False
 
+        # informal constants used in parse(cob_id, data, eds)
+        self.SDO_tx_min = 0x580
+        self.SDO_rx_min = 0x600
+        self.SDO_rx_max_plus_1 = 0x680
+
```
Comment (thaotran9) [inline canopen_monitor/parse/sdo.py:887]: these ranges are in message.py, I think we can reuse them instead of creating new fields for this class.

Comment (TheDragonMask) [reply canopen_monitor/parse/sdo.py:887]: Good point, I will try to figure out how to import then from messages.py after our meeting with Casamento then.

**Comment by TheDragonMask** [comment]
Came across a weird error when trying to "amend" my previous commit, and since then github doesn't like this pull-request. I've also made a mess of things by trying to revert the commit several times, and at this point it is simpler for me to just make another one pull-request that makes the same changes. In the interest of keeping this one as a record, I am closing it with this comment

**Pull request closed without merging**